perun-wui-registrar/src/main/java/cz/metacentrum/perun/wui/registrar/widgets/items/Username.java

```diff
@@ -29,6 +29,16 @@
 	private final UsernameValidator validator;
 
 	private InputGroup widget;
+	private ExtendedTextBox box;
+	private boolean enabled = true;
+
+	@Override
+	public void setEnabled(boolean enabled) {
```

Comment:
There is former method: `public void setEnable(boolean enable)`. You should check its usage/impl and probably merge it with the new (more generic) `setEnabled()` method.

Reply:
Done.